go.mod

```diff
@@ -1,6 +1,6 @@
-module github.com/grafana/agent-remote-config
+module github.com/grafana/alloy-remote-config
```

Comment:
I assume we did this to not break things but here I am breaking things so let's clean this up now I think